kie-server-parent/kie-server-services/kie-server-services-common/pom.xml

```diff
@@ -15,8 +15,46 @@
 
   <properties>
     <java.module.name>org.kie.server.services.common</java.module.name>
+    
+    <io.fabric8.kubernetes-client.version>4.1.0</io.fabric8.kubernetes-client.version>
```

Comment:
Version of Kubernetes client is defined in parent pom - https://github.com/kiegroup/droolsjbpm-build-bootstrap/blob/master/pom.xml
Please if you need to raise the version, raise it there so it is unified across all projects.

Reply:
Since it is located at different repository, do we need separate new PR and or JIRA for this POM change of raising the fabric8 client version?

Reply:
I think you can reuse existing JIRA for this. The new PR needs to be created in affected repo, if you use the same branch name in that repo then those changes will be reused in this PR.

Reply:
I'd prefer to use existing JIRA. The change has been committed into forked repo with the same name of feature branch. (See below)
https://github.com/rhtevan/droolsjbpm-build-bootstrap/tree/kie-server-ocp-rearch-1

Is there anything else I need to do with this?

Reply:
Please make a PR in droolsjbpm-build-bootstrap and reference this PR there.

Reply:
Created a new PR in droolsjbpm-build-bootstrap.

https://github.com/kiegroup/droolsjbpm-build-bootstrap/pull/875

Reply:
@sutaakar Raise the fabric8 client version at 'build-bootstrap' module breaks KIE-WB build. Shall we revert it back, or raise a new issue to fix KIE-WB build?

Reply:
The best approach would be to fix KIE-WB so we have the only version of fabric8 used everywhere.

Reply:
Retested with current master branch code, the broken build issue with KIE-WB-COMMON-ALA has been fixed.